account_invoice_comment_template/tests/test_account_invoice_report.py

```diff
@@ -68,7 +68,6 @@ def test_comments_in_invoice_report(self):
         self.assertRegexpMatches(str(res[0]), self.after_comment.text)
 
     def test_comments_in_invoice(self):
-        self.partner.property_comment_template_id = self.after_comment.id
```

Comment:
Shouldn't you assign `base_comment_template_ids` instead?
```suggestion
    def test_comments_in_invoice(self):
        self.partner.base_comment_template_ids = [(4, self.after_comment.id)]
```

Reply:
Is not necessary because they have already been defined in the setup test.